package.json

```diff
@@ -16,37 +16,38 @@
   },
   "dependencies": {
     "date-fns": "^2.21.1",
-    "expo": "^38.0.0",
-    "expo-barcode-scanner": "~8.2.1",
-    "expo-camera": "~8.3.1",
-    "expo-constants": "~9.1.1",
-    "expo-crypto": "~7.0.0",
-    "expo-linear-gradient": "~8.2.1",
-    "expo-linking": "^1.0.3",
-    "expo-localization": "^8.2.1",
-    "expo-permissions": "~9.0.1",
-    "expo-updates": "^0.2.12",
+    "expo": "^42.0.0",
+    "expo-barcode-scanner": "~10.2.2",
+    "expo-camera": "~11.2.1",
+    "expo-constants": "~11.0.1",
+    "expo-crypto": "~9.2.0",
+    "expo-device": "^3.3.0",
+    "expo-linear-gradient": "~9.2.0",
+    "expo-linking": "~2.3.1",
+    "expo-localization": "~10.2.0",
+    "expo-updates": "~0.8.0",
     "fp-ts": "^2.9.5",
     "google-libphonenumber": "^3.2.19",
     "io-ts": "^2.2.16",
     "io-ts-reporters": "^1.2.2",
     "io-ts-types": "^0.5.16",
     "lodash": "^4.17.21",
-    "react": "16.11.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz",
-    "react-native-gesture-handler": "~1.8.0",
+    "react": "16.13.1",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
+    "react-native-gesture-handler": "~1.10.2",
     "react-native-qrcode-svg": "^6.0.3",
     "react-native-reanimated": "~1.9.0",
-    "react-native-screens": "~2.9.0",
-    "react-native-svg": "12.1.0",
-    "react-native-webview": "^11.4.3",
+    "react-native-screens": "~3.4.0",
+    "react-native-svg": "12.1.1",
+    "react-native-webview": "11.6.2",
     "react-navigation": "^4.1.0",
     "react-navigation-drawer": "^2.5.0",
     "react-navigation-stack": "^1.10.3",
-    "sentry-expo": "^2.0.3"
+    "sentry-expo": "^4.0.0",
+    "unimodules-permissions-interface": "^6.1.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.13.16",
+    "@babel/core": "~7.9.0",
```

Comment:
Why is babel downgraded?

Reply:
as discussed, will update `@babel/core` to `^7.13.16`